scripts/run.py

```diff
@@ -302,15 +303,29 @@ def get_scene(scene):
 
 		resolution = [args.width or 1920, args.height or 1080]
 		n_frames = args.video_n_seconds * args.video_fps
+		save_frames = "%" in args.video_output
+		start_frame, end_frame = args.video_render_range
 
 		if "tmp" in os.listdir():
 			shutil.rmtree("tmp")
 		os.makedirs("tmp")
 
+		if start_frame > 0:
+			# When not rendering from the beginning, first rendered frame will have excessive motion blur, so we discard it
+			testbed.render(resolution[0], resolution[1], args.video_spp, True, 0.0, float(start_frame)/n_frames, args.video_fps, shutter_fraction=0.5)
```

Comment:
Could you change this to use `float(start_frame-1)/n_frames`? I don't think it'll have a visible impact in the current state of the code, but logically this line should set the internal state of the rendering process to the frame *before* rendering starts.

Reply:
Fixed in fa7272627081a0213ba5120101f0762dcf80bf3c